modules/appeals_api/spec/models/notice_of_disagreement_spec.rb

```diff
@@ -86,6 +86,17 @@
     end
   end
 
+  context 'Veteran birth date is not in the future' do
+    it 'using a birth date /not/ in the past creates an invalid record' do
```

Comment:
This can be shortened to `creates an invalid record` since the context takes care of the first part of the statement